src/main/java/xyz/aadev/generitech/common/tileentities/power/TileEntityPowerStorage.java

```diff
@@ -32,63 +32,41 @@
 import net.minecraftforge.common.capabilities.Capability;
 import xyz.aadev.aalib.common.inventory.InternalInventory;
 import xyz.aadev.aalib.common.inventory.InventoryOperation;
-import xyz.aadev.generitech.client.gui.power.GuiPowerStorage;
-import xyz.aadev.generitech.common.container.power.ContanierPowerStorage;
+import xyz.aadev.aalib.common.tileentities.TileEntityBase;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'xyz.aadev.aalib.common.tileentities.TileEntityBase'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)